proto/common.proto

```diff
@@ -108,6 +108,13 @@ enum Stat {
 	StatFrostResistance = 31;
 	StatNatureResistance = 32;
 	StatShadowResistance = 33;
+	StatRunicPower = 34;
+	StatBloodRune = 35;
+	StatFrostRune = 36;
+	StatUnholyRune = 37;
+	StatDeathRune = 38;
+	StatWeaponDamageMin = 39;
```

Comment:
Rather than 2 separate stats, I think you probably want `weapon_dps` or weapon_damage`. I can't think of any scenario where you'd want to increase one without the other.